src/static/js/pluginfw/plugins.js

```diff
@@ -28,17 +27,17 @@ exports.formatPluginsWithVersion = function () {
   return plugins.join(', ');
 };
 
-exports.formatParts = function () {
-  return _.map(defs.parts, (part) => part.full_name).join('\n');
-};
+exports.formatParts = () => _.map(defs.parts, (part) => part.full_name).join('\n');
 
-exports.formatHooks = function (hook_set_name) {
+exports.formatHooks = (hook_set_name) => {
   const res = [];
   const hooks = pluginUtils.extractHooks(defs.parts, hook_set_name || 'hooks');
 
   _.chain(hooks).keys().forEach((hook_name) => {
     _.forEach(hooks[hook_name], (hook) => {
-      res.push(`<dt>${hook.hook_name}</dt><dd>${hook.hook_fn_name} from ${hook.part.full_name}</dd>`);
+      res.push(`<dt>${hook.hook_name}</dt>
+        <dd>${hook.hook_fn_name} from ${hook.part.full_name}</dd>
+        `);
```

Comment:
Don't introduce newlines into the string.

```suggestion
      res.push(`<dt>${hook.hook_name}</dt><dd>${hook.hook_fn_name} ` +
               `from ${hook.part.full_name}</dd>`);
```